config_example.json

```diff
@@ -15,19 +15,146 @@
         "fennec_nightly_repacks": "/path/to/beetmoverscript/beetmoverscript/templates/fennec_nightly_repacks.yml",
         "fennecx86_nightly": "/path/to/beetmoverscript/beetmoverscript/templates/fennecx86_nightly.yml"
     },
+
     "s3": {
         "firefox": {
-            "bucket": "mozilla-releng-beetmoverscript-firefox",
-            "credentials": {
-                "id": "dummy",
-                "key": "dummy"
+            "nightly": {
+                "push-to-nightly": {
+                    "s3_prefixes": {
+                        "default": {
+                            "s3_prefix": "pub/firefox/nightly/{}-{}/",
```

Comment:
Using `"{}".format(var)` is handy, but it means we have to know order.

We can also use format with kwargs, or % strings with kwargs, which allows us to replace without having to know or care about order:
```
>>> "{}".format("a")
'a'
>>> "{two}{one}".format(one=1, two=2)
'21'
>>> "%(two)s %(one)s" % {"one": 1, "two": 2}
'2 1'
```

That also allows us to have a generic dictionary to replace things with.  For example, 

```
>>> "{two}{one}".format(one=1, two=2, three=3)
'21'
>>> "%(two)s %(one)s" % {"one": 1, "two": 2, "three": 3}
'2 1'
```
still work even though `three` isn't in the template.